doc/tutorials/run_getting_started.py

```diff
@@ -0,0 +1,71 @@
+""".. _getting-started-tutorial:
```

Comment:
I don't know where this is relevant. Should this be included? I basically took it over from the tutorial `.py` files.

Reply:
This is just adding an [explicit target label](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets) so that the document can be referenced using it. Is a nice to have but feel free to remove it if it finds no use.

Reply:
Okay, it's not used now but I will just leave it there as it might be useful in the future (and it doesn't hurt to keep it)